pyproject.toml

```diff
@@ -1,7 +1,168 @@
 [build-system]
-requires = ["setuptools>=67", "setuptools_scm[toml]>=7.0"]
+requires = [
+    "setuptools==69.0.0",
+    "setuptools_scm==8.0.4",
+]
 build-backend = "setuptools.build_meta"
 
+[project]
+dynamic = ["version"]
+name = "briefcase"
+description = "Tools to support converting a Python project into a standalone native application."
+readme = "README.rst"
+requires-python = ">= 3.8"
+license.file = "LICENSE"
+authors = [
+    {name="Russell Keith-Magee", email="russell@keith-magee.com"},
+]
+maintainers = [
+    {name="BeeWare Team", email="team@beeware.org"},
+]
+keywords = [
+    "app",
+    "packaging",
+    "installer",
+    "macOS",
+    "iOS",
+    "android",
+    "mobile",
+    "windows",
+    "tvOS",
+    "watch",
+    "watchOS",
+    "wearos",
+    "web",
+    "pyscript",
+    "pyodide",
+]
```

Comment:
no linux... 😢 

but more seriously....adding Flatpak (and even AppImage if we dare) may be a good idea....going further, deb, RPM, pkg....less committed to these, though

Reply:
O_O 

No idea how linux fell off this list. But yes, will add Flatpak et al.